packages/cli/cli/src/commands/validate/logViolations.ts

```diff
@@ -26,44 +27,79 @@ export function logViolations({
         logViolationsSummary({ context, stats, logWarnings });
     }
 
-    violations.forEach((violation) => {
-        if (violation.severity === "fatal" || violation.severity === "error") {
-            logViolation({ violation, context });
-        }
-    });
+    // Group all violations by nodePath
```

Comment:
nit: Redundant comment.

Reply:
I use AI to generate ONE thing 😭 (fixed)